apps/functions/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("//tools:defaults.bzl", "esbuild", "ts_library")
+load("@build_bazel_rules_nodejs//:index.bzl", "copy_to_bin")
+
+package(default_visibility = ["//visibility:private"])
+
+copy_to_bin(
+    name = "functions_files",
+    srcs = [
+        "package.json",
+    ],
+    visibility = ["//apps:__pkg__"],
+)
+
+ts_library(
+    name = "functions",
+    srcs = [
+        "index.ts",
+    ],
+    deps = [
+        "//apps/functions/githubWebhook",
+        "@npm//firebase-functions",
+    ],
+)
+
+esbuild(
+    name = "functions_compiled",
+    entry_points = [
+        "index.ts",
+    ],
+    format = "cjs",
```

Comment:
Should we use `ESM` here? looks like firebase tools now supports this and long-term we want to use ESM everywhere. Doing it now would make the migration easier in the future then (CJS -> ESM is never easy)

https://github.com/firebase/firebase-tools/releases/tag/v9.15.0